templates/tsconfig.json.tmpl

```diff
@@ -4,7 +4,7 @@
 		"declaration": true,
 		"outDir": "dist-node",
 		"rootDir": "./src",
-		"types": ["node", "mocha", "sinon", "chai"]
-	},
-	"include": ["src/**/*", "types/**/*"]
+		"types": ["node", "jest", "jest-extended"]
```

Comment:
this file shouldnt be changed yet, as still some package depends on it.
For now, it symlink should be removed and duplicated